projects/suricata/build.sh

```diff
@@ -91,6 +91,14 @@ then
     fuzz_branches+=("6")
 fi
 
+if [[ `shuf -i 0-9 -n 1` -eq 0 ]]
```

Comment:
is there a better way such as an environment variable ? cc @Alan32Liu

Reply:
Maybe try `CFLAGS` and `CXXFLAGS`?
Here is [a similar example](https://github.com/google/oss-fuzz/blob/e8d9b9749abd171b4606def48b9648ec44f517af/infra/base-images/base-builder/compile_centipede#L27C95-L27C125).

Adding it to `LIB_FUZZING_ENGINE` is indeed too hacky.